### ISSUE TITLE
Bump `quill_flutter` to `^5.0.0`

### ISSUE THREAD

**Issue by harkairt** [friebetill/delta_markdown#8]
Till then workaround if anyone needs it:

```
  delta_markdown:
    git: 
      url: git@github.com:harkairt/delta_markdown.git
```

**Comment by bitelligent** [comment]
please make it public @harkairt

**Comment by harkairt** [comment]
@bitelligent it is supposed to be public. can you clone it?

**Comment by bitelligent** [comment]
No I'm using it in pubspec.yaml like this @harkairt 

delta_markdown:
    git:
      url: git@github.com:harkairt/delta_markdown.git

**Comment by bitelligent** [comment]
Help me to resolve this please @harkairt

**Comment by harkairt** [comment]
hmm, it seems that the syntax I linked was wrong. it should probably be
```
  delta_markdown:
    git: 
      url: https://github.com/harkairt/delta_markdown.git
```

**Comment by friebetill** [comment]
Thanks @harkairt for your PR and help! I've released the new version `0.7.0`. 
```yaml
  delta_markdown: ^0.7.0
```
